Maestro/subscriptions.json

```diff
@@ -30,11 +30,11 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 3159
     },
-    // A build definition that will trigger the dotnet-nightly Docker image build in Docker Hub
-    "build-dotnet-docker-nightly": {
+    // A build definition capable of running any .ps1 script in the dotnet-docker-nightly repo
+    "dotnet-docker-nightly-general": {
       "vsoInstance": "mseng.visualstudio.com",
       "vsoProject": "dotnetcore",
-      "buildDefinitionId": 3590
```

Comment:
@eerhardt, yes we are going to delete the old build def.  We had created another build definition already so that we could update the dependencies and test everything out without breaking the existing Maestro hook.  It was a good thing we went down this approach because @naamunds encountered a couple issues that needed to be flushed out.